more/free-programming-cheatsheets.md

```diff
@@ -159,6 +159,7 @@
 ### GraphQL
 
 * [GraphQL Cheatsheet](https://devhints.io/graphql) - devhints, Rico Santa Cruz (HTML)
+* [GraphQL Cheat Sheet](https://simplecheatsheet.com/tag/graphql-cheat-sheet) - Simple Cheat Sheet (HTML)
```

Comment:
Alphabetically order the entries. Here swap line 161 and 162

Reply:
Solved